pkg/api/unversioned.go

```diff
@@ -29,6 +29,19 @@ type APIVersions struct {
 	Versions []string `json:"versions"`
 }
 
+// APIResource stores the name of a resource and whether it is namespaced.
+type APIResource struct {
+	Resource   string `json:"resource"`
```

Comment:
I'm wondering whether we should expose "kind" for both request and response now, or let that stay swagger.  I think we want to have input Kind so that a client can search for the resource to post an arbitrary object from create -f -.  Kind would only be set if this is the canonical "resource", not secondary resources that perform specific actions.

We should definitely resist duplicating swagger as much as possible (in the future we can link directly to the swagger doc for this resource endpoint).